fuzz/fuzz_targets/uint128_fuzz.rs

```diff
@@ -0,0 +1,101 @@
+extern crate bitcoin;
+use std::str::FromStr;
+
+fn do_test(data: &[u8]) {
+    macro_rules! read_ints {
+        ($start: expr) => { {
+            let mut native = 0;
+            for c in data[$start..$start + 16].iter() {
+                native <<= 8;
+                native |= (*c) as u128;
+            }
+            // Note BE:
+            let uint128 = bitcoin::util::uint::Uint128::from(&[native as u64, (native >> 8*8) as u64][..]);
+            (native, uint128)
+        } }
+    }
+    macro_rules! check_eq {
+        ($native: expr, $uint: expr) => { {
+            assert_eq!(&[$native as u64, ($native >> 8*8) as u64], $uint.as_bytes());
```

Comment:
`as_bytes` returning `[u64;2]` is quite irritating. Do you know if there's a good rationale for calling it that? Might open a PR otherwise to fix it.

Reply:
Ha, no, thats super confusing. Agreed.